include/deal.II/base/vectorization.h

```diff
@@ -224,6 +224,14 @@ class VectorizedArray
   // POD means that there should be no user-defined constructors, destructors
   // and copy functions (the standard is somewhat relaxed in C++2011, though).
 
+  VectorizedArray() = default;
+
+  // Construct an array with the given scalar broadcast to all lanes
```

Comment:
This is not a `doxygen` comment. Can you please do
```suggestion
  /**
   * Construct an array with the given scalar broadcast to all lanes.
   */
```
here and below?

Reply:
I have changed these lines.